libraries/SITL/SIM_Webots.cpp

```diff
@@ -487,7 +486,7 @@ void Webots::update(const struct sitl_input &input)
         position.xy() += origin.get_distance_NE_double(home);
 
         // limit to 16G to match pixhawk1
```

Comment:
Comment is now wrong.

Reply:
> > Webots 2021 files are kept on a separate sub-folder.
> > Webots 2023 files are grouped in another subfolder
> > so future releases can be added as folders in the example Webots folder
> 
> Is there some reason we need to keep the 2021 version folder around? It will increase checkout sizes and lose history on things that have been copied across to the new folder.
> 
> We're talking ~1MB of copied stuff.

The new files are not backward compatible. That is why I thought to leave the files just in case someone has build code over the old version.

Anyway it is your call.